api/commands/run.ts

```diff
@@ -93,10 +93,10 @@ export class Runner {
   }
 
   private triggerChange() {
-    this.changeListeners.forEach(listener => listener(this.result));
+    return Promise.all(this.changeListeners.map(listener => listener(this.result)))
```

Comment:
please change `onChange` to be: `public onChange(listener: (graph: dataform.IExecutedGraph) => Promise<void> | void): Runner`

Reply:
added: https://github.com/dataform-co/dataform/pull/218/files#diff-f23e82029447c823a22994252f66fa60R51